tests/behaviour/test_stilyagi_install_steps.py

```
@@ -0,0 +1,196 @@
+@when("I run stilyagi install with an auto-discovered version")
+def run_install_auto(
+    repo_root: Path,
+    external_repo: Path,
+    monkeypatch: pytest.MonkeyPatch,
+    scenario_state: dict[str, object],
+) -> None:
+    """Invoke install without explicit version, relying on release discovery."""
+    import concordat_vale.stilyagi as stilyagi_module
+    import concordat_vale.stilyagi_install as install_module
+
+    def fake_fetch_latest_release(repo: str) -> dict[str, object]:
```

Ruff (0.14.5)

Unused function argument: `repo`

(ARG001)